src/clj/chronograph/db/users.clj

```diff
@@ -0,0 +1,89 @@
+(ns chronograph.db.users
+  (:require [next.jdbc :as jdbc]
+            [next.jdbc.sql :as sql]
+            [next.jdbc.result-set :as jdbc-result-set]
+            [next.jdbc :as jdbc]
+            [chronograph.db.core :as db]
+            [camel-snake-kebab.core :as csk])
+  (:import (java.time Instant)))
+
+(def ^:private sql-opts {:builder-fn jdbc-result-set/as-unqualified-kebab-maps
```

Comment:
These options would be common across namespaces no? Can we move them to `c.db.core`?

Reply:
Done